js/src/api/pubsub/eth/eth.js

```diff
@@ -0,0 +1,236 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+import PubsubBase from '../pubsubBase';
+
+import { inAddress, inBlockNumber, inHex, inNumber16, inOptions, inFilter } from '../../format/input';
+import { outAddress, outBlock, outNumber, outTransaction, outSyncing, outReceipt, outLog } from '../../format/output';
+
+export default class Eth extends PubsubBase {
+  constructor (transport) {
+    super(transport);
+    this._api = 'parity';
+  }
+
+  newHeads (callback) {
+    return this.addListener('eth', 'newHeads', callback);
+  }
+  //
+  // logs (callback) {
```

Comment:
Logs are now available: #5705 but this can come in a separate PR.

Imho we should either uncomment the code or remove it.